ndmg/scripts/ndmg_dwi_pipeline.py

```diff
@@ -558,16 +604,11 @@ def ndmg_dwi_worker(
         elif reg_style == "native":
             # align atlas to t1w to dwi
             print("%s%s" % ("Applying native-space alignment to ", labels[idx]))
-            labels_im_file = gen_utils.reorient_img(labels[idx], namer)
-            labels_im_file = gen_utils.match_target_vox_res(
-                labels_im_file, vox_size, namer, sens="t1w"
-            )
-            labels_im_file_dwi = reg.atlas2t1w2dwi_align(labels_im_file, dsn=False)
-            labels_im = nib.load(labels_im_file_dwi)
+            labels_im = nib.load(labels_im_file_dwi_list[idx])
             g1 = gen_graph.graph_tools(
                 attr=len(np.unique(np.around(labels_im.get_data()).astype("int16")))
                 - 1,
-                rois=labels_im_file_dwi,
+                rois=labels_im_file_dwi_list[idx],
```

Comment:
can you explain this change briefly? I guess this is all happening inside a loop now? (hard to see the overall structure change on github)